connector_importer/models/job_mixin.py

```diff
@@ -2,17 +2,16 @@
 # Copyright 2018 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import _, api, exceptions, fields
+from odoo import _, api, exceptions, fields, models
 
 from odoo.addons.queue_job.job import DONE, STATES
 
 
-class JobRelatedMixin(object):
-    """Mixin klass for queue.job relationship.
+class JobRelatedMixin(models.AbstractModel):
+    """ Mixin klass for queue.job relationship. """
 
-    We do not use an abstract model to be able to not re-define
-    the relation on each inheriting model.
-    """
+    _name = "job.related.mixin"
```

Comment:
not used anywhere apparently :thinking:

Reply:
Good day @simahawk, `ImportRecord` and `ImportRecordSet` are using it. Please advise if anything need to be changed.
https://github.com/OCA/connector-interfaces/blob/3c9d462a3ef024f7415206d53adfb0c7a7d35d27/connector_importer/models/record.py#L17
https://github.com/OCA/connector-interfaces/blob/3c9d462a3ef024f7415206d53adfb0c7a7d35d27/connector_importer/models/recordset.py#L18

Reply:
yes but that's a normal python inheritance which does not need the mixin to inherit from the abstract model.
If you define it as an abstract model I would expect to be used for inheritance in the odoo way.
When I worked on this back in the days there was something that was not working properly.
I'll check.